packages/terra-dev-site/src/pages/page/layouts/Card.module.scss

```diff
@@ -16,7 +16,7 @@
     flex-direction: column;
     height: calc(100% - 30px);
     min-height: calc(100% - 30px);
-    overflow: hidden;
+    overflow: auto;
```

Comment:
Thanks @sambao21 - apologies, we actually resolved the issue already with one of our recent PRs #316, which fixed the issues with both clipping and scrolling. But in the rush to get it out for the consumers needing it patched-back for dev-site@v7, we then forgot to apply the changes also to main for the latest and dev-site@v8.  

If you could apply the same changes from #316, that would be the design approach we would like to resolve the issue -- where the it is the page, with a content-sized card on it, overflows and scrolls. Not allowing the scrolling to be on the interior of the card as this change would make.